app/enterprise/1.3-x/admin-api/audit-log.md

```diff
@@ -190,6 +190,19 @@ The following request paths generate an audit log entry in the database:
 
 [Back to TOC](#table-of-contents)
 
+### Audit Log Retention
+
+Request audit records will be kept in the database for a duration define by the
```

Comment:
Suggest "Request audit records are kept in the database for a period of time defined by the"